lib/rails_param/param_evaluator.rb

```diff
@@ -48,6 +46,8 @@ def param!(name, type, options = {}, &block)
     private
 
     def recurse_on_parameter(parameter, &block)
+      return if parameter.value.nil?
```

Comment:
No point recursing and checking nested params, if the `parameter` itself is `nil`.